index.ts

```diff
@@ -13,28 +21,46 @@ export * from '@node-redis/json';
 export * from '@node-redis/search';
 export * from '@node-redis/time-series';
 
-const modules =  {
+const modules = {
     ...RedisBloomModules,
     graph: RedisGraph,
     json: RedisJSON,
     ft: RediSearch,
     ts: RedisTimeSeries
 };
 
-export function createClient<S extends RedisScripts>(
-    options?: Omit<RedisClientOptions<never, S>, 'modules'>
-): RedisClientType<typeof modules, S> {
+export type RedisDefaultModules = typeof modules;
+
+export type RedisClientType<
```

Comment:
I don't have experience with this, but is this not running into conflicts with `export * from '@node-redis/client';` where `RedisClientType` is also exported?

Reply:
No, it'll override it